keyboards/splitkb/aurora/corne/keymaps/x123/config.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 x123 <@x123>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define ONESHOT_TAP_TOGGLE 3  /* Tapping this number of times holds the key until tapped once again. */
+#define ONESHOT_TIMEOUT 5000  /* Time (in ms) before the one shot key is released */
+
+#define TAPPING_TOGGLE 1
+#define TAPPING_TERM 280
+#define IGNORE_MOD_TAP_INTERRUPT
+#define UNICODE_SELECTED_MODES UNICODE_MODE_WINCOMPOSE, UNICODE_MODE_MACOS, UNICODE_MODE_LINUX
+
+#define MASTER_LEFT
```

Comment:
This isn't actually needed, as master left is the default behavior, if no handedness defines are  present. 

```suggestion
```

Reply:
Removed.